test/BUILD

```diff
@@ -1,6 +1,21 @@
 package(default_visibility = ["//visibility:public"])
 
-load("//bazel:envoy_build_system.bzl", "envoy_cc_test_library")
+load("//bazel:envoy_build_system.bzl", "envoy_cc_test", "envoy_cc_test_library")
+
+envoy_cc_test(
+    name = "example_configs_test",
+    srcs = [
+        "example_configs_test.cc",
+        "example_configs_test.h",
+    ],
+    data = ["//configs:example_configs"] + ["example_configs_test_setup.sh"],
```

Comment:
Here's my Bazel version:

```
Build label: 0.4.5
Build target: bazel-out/local-fastbuild/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Thu Mar 16 12:19:38 2017 (1489666778)
Build timestamp: 1489666778
Build timestamp as int: 1489666778
```

It could be a function of kernel config, since sandboxing is built on the underlying namespaces mechanisms, so they need to be there for it to work.